<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pag_Comentario</class>
 <widget class="QMainWindow" name="Pag_Comentario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Perfil_User</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_desc">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>93</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_like">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Like</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_comentario">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_comentar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Comentar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_share">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Compartir</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listBox_coment">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
